src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -110,6 +110,14 @@ const OVERRIDE_SENSOR_UNITS = {
   pressure: ["hPa", "Pa", "kPa", "bar", "cbar", "mbar", "mmHg", "inHg", "psi"],
 };
 
+const OVERRIDE_WEATHER_UNITS = {
+  precipitation: ["°C", "°F", "K"],
+  pressure: ["hPa", "mbar", "mmHg", "inHg"],
+  temperature: ["°C", "°F"],
+  visibility: ["km", "mi"],
+  wind_speed: ["km/h", "mph", "m/s"],
```

Comment:
```suggestion
  wind_speed: ["km/h", "mph", "m/s", "ft/s", "kt"],
```

Reply:
Support for knot may make sense. Where is ft/s used though?